platform/cosmos/api.go

```diff
@@ -101,19 +101,30 @@ func (p *Platform) GetDelegations(address string) (blockatlas.DelegationsPage, e
 	if err != nil {
 		return nil, err
 	}
-
-	results = append(results, NormalizeDelegations(delegations)...)
-	results = append(results, NormalizeUnbondingDelegations(unbondingDelegations)...)
+	validatorList, err := services.GetValidators(p)
+	if err != nil {
+		return nil, err
+	}
+	validators := make(blockatlas.ValidatorMap)
+	for _, v := range validatorList {
+		validators[v.ID] = v
+	}
+	results = append(results, NormalizeDelegations(delegations, validators)...)
+	results = append(results, NormalizeUnbondingDelegations(unbondingDelegations, validators)...)
 
 	return results, nil
 }
 
-func NormalizeDelegations(delegations []Delegation) []blockatlas.Delegation {
+func NormalizeDelegations(delegations []Delegation, validators blockatlas.ValidatorMap) []blockatlas.Delegation {
 	c := coin.Cosmos()
 	results := make([]blockatlas.Delegation, 0)
 	for _, v := range delegations {
+		validator, ok := validators[v.ValidatorAddress]
+		if !ok {
+			logger.Error("Validator not found", validator)
```

Comment:
do you need to continue?

Reply:
You know more than me, can you help me with this question? Do we need to continue? I think we don't have a problem. We can show just the JSON below in this case.

```
[
   {
      "delegator":{
         "id":"cosmosvaloper1qwl879nx9t6kef4supyazayf7vjhennyh568ys",
      },
      "coin":{
         "coin":118,
         "symbol":"ATOM",
         "name":"Cosmos ATOM",
         "decimals":6
      },
      "value":"3000000",
      "status":"active"
   }
]
```

Reply:
i don't think it make sense to return this way. if validated not found with information we should not present it here.

Reply:
we should only present validators that provided information in assets repo

Reply:
great. I going to fix this. Thks